<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DynaLog Rekonstruktion</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="box_dirs">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>331</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Verzeichnisse</string>
    </property>
    <widget class="QLineEdit" name="edit_dicomdir">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_dicomdir">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>DICOM Dateien:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_dynadir">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>DynaLog Dateien:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_outputdir">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Ausgabe:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_dynadir">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_outputdir">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="button_dicomdir">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>25</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="button_dynadir">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>50</y>
       <width>25</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="button_outputdir">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>25</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="box_plans">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>781</width>
      <height>221</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="title">
     <string>Erkannte Pl채ne</string>
    </property>
    <widget class="QTableWidget" name="table_plans">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>761</width>
       <height>161</height>
      </rect>
     </property>
     <property name="columnCount">
      <number>4</number>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Patient</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Planname</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>DynaLog komplett</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Plan UID</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="button_plans_refresh">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>190</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Aktualisieren</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_onlyarcslisted">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>291</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Es werden nur Pl채ne mit mindestens einem Arc aufgef체hrt!</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="box_export">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>401</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Export</string>
    </property>
    <widget class="QPushButton" name="button_export">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>DynaLog-Pl채ne
erstellen</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressbar_export">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>261</width>
       <height>21</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QProgressBar" name="busybar">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>261</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="textVisible">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Optionen</string>
    </property>
    <addaction name="actionExport"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
